app/scripts/lib/experiment.js

```diff
@@ -157,16 +159,17 @@ define(function (require, exports, module) {
      * Get the experiment group for `experimentName` the user is in.
      *
      * @param {String} experimentName
+     * @param {Object} [additionalInfo] additional info to pass to Able.
```

Comment:
We only have a handful of `optional={}` in this repo:

```bash
➜  scripts git:(additional-args-to-experiments) ✗ rgrep "options={}" .                                                       git:(additional-args-to-experiments|…
./views/marketing_snippet.js:     * @param {Object} [options={}]
./views/mixins/marketing-mixin.js:   * @param {Object} [options={}] options
./views/mixins/resend-mixin.js:   * @param {Object} [options={}] options
```

All of these have named optional parameters declared afterwards.